layouts/_default/baseof.html

```diff
@@ -9,6 +9,7 @@
     <header>
       {{ partial "navbar.html" . }}
     </header>
+    {{ partial "notification.html" }}
```

Comment:
Looks good! But it's not site-wide. For example, not on the docs pages. Is that intended?